### DISCUSSION TITLE
How to use 'OutputSelect'?

### DISCUSSION THREAD

**Discussion by sghqsads** [CURENT/andes#474] [Q&A] [answered]
Hello everyone,

When I was using 'OutputSelect', I created a new subsheet named 'Output' for my input file and filled in 'model, varname and dev', for instance:

model      varname   dev
GENROU   Pe            GENROU_2

However, when I started to run Andes, it pops out that 'ValueError: Mandatory parameter model for Output missing'.

I was wondering how 'OutputSelect' could be used properly? Thanks.

**Comment by jinningwang** [comment]
I'll do a demonstration recently to show this. Stay tuned.

**Comment by cuihantao** [comment, marked as answer]
It turns out to be a bug in parsing the day input file. If the Output or other sheets do not have a uid column, the Xlsx parser will fail.

It has been fixed in the latest release. I’m

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by sghqsads** [comment]
Thank you!
